master/buildbot/db/logs.py

```diff
@@ -346,7 +346,7 @@ def thdcompressLog(conn):
         return saved
 
     # returns a Deferred that returns a value
-    def deleteOldLogChunks(self, older_than_timestamp):
+    def deleteOldLogChunks(self, older_than_timestamp, exceptions=[]):
```

Comment:
Shouldn't default value of `exceptions` be an empty dict?

Reply:
Same as explained above. In the LogChunksJanitor class, only the names are passed. Maybe passing a dict and just using the names is better. What do you think?

Reply:
Initializing parameters with mutable objects is strongly discouraged. https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments